assets/css/prism.css

```diff
@@ -1,5 +1,5 @@
-/* PrismJS 1.22.0
```

Comment:
Can we have the Prism stuff in another PR?

Reply:
can, but I put it in here because I added new syntax highlighting necessary for this post (docker files) so we'd have to merge it first.
what's the value of splitting the PRs tho?

Reply:
Having two different commits once we squash the PRs

Reply:
I split out the prism commit, at this point I don't think we have to squash the PR, because all 3 commits are stand alone & self contained (but build upon each other)

* default avatar
* prism
* article

if you prefer I can open 2 more PRs for the individual changes

Reply:
moved the prism stuff over into #23 so it doesn't create conflicts with additional syntax highlighting required for other posts. the diff still shows up here because this branch is on top of the other one, so the local preview renders code snippets properly.